spec.html

```diff
@@ -4276,16 +4292,16 @@ <h1>GetValue ( _V_ )</h1>
         <emu-alg>
           1. ReturnIfAbrupt(_V_).
           1. If Type(_V_) is not Reference, return _V_.
-          1. Let _base_ be GetBase(_V_).
           1. If IsUnresolvableReference(_V_) is *true*, throw a *ReferenceError* exception.
+          1. Let _base_ be _V_.[[Base]].
```

Comment:
Any reason this step was moved? It's not observable, but curious.

Reply:
In order to avoid `base` being `~empty~`; 3a705d9 moves other occurrences.